### DISCUSSION TITLE
No ttf Folder

### DISCUSSION THREAD

**Discussion by GWevroy** [tonsky/FiraCode#1292] [General]
Hi, 
I have installed Fira Code several times now, but not for some time. I downloaded it again this evening, but when I look for the ttf fonts to install in Windows, I can't find them. The README file is vague and just says "look in the ttf folder" but it doesn't exist. I did find a script in there that I executed, but that just creates a blank ttf sub-folder.

So where have the ttf fonts gone for Windows?
PS. Thank you for an excellent font set. I use these on a multitude of computers both at work and at home.

**Comment by alexeyten** [comment]
You’ve probably downloaded repository. I guess you should use fonts from [release page](https://github.com/tonsky/FiraCode/releases/tag/5.2)
